src/plugins/blocksmith/components/article-block-preview/index.jsx

```diff
@@ -21,13 +22,18 @@ const ArticleBlockPreview = ({ className, articleNode, imageNode, block }) => {
         target="_blank"
       >
         {
-          block.get('show_image') ? ([
-            <img key="a" src={imgSrc} alt="thumbnail" />,
-            <div key="b" className="launch-text">
-              <p>{block.get('link_text') || articleNode.get('title')}</p>
-              <Icon alert border imgSrc="book-open" size="xs" alt="article-icon" />
-            </div>,
-          ]) : (
+          block.get('show_image') ? (
+            <div className="image-wrapper">
```

Comment:
@bhsiao55 - does the _image-wrapper_ class apply any styles? If you added it as a wrapper tag, then you can wrap it with the empty tags like this: `<>blah</>`